layer/logged_data/loggers.py

```diff
@@ -0,0 +1,357 @@
+from argparse import Namespace
```

Comment:
Can we make this a file named `pytorch_lightning.py` under the `loggers/` directory?

Reply:
@cihati won't like it but done 😈